src/evaluators.js

```diff
@@ -15,7 +15,7 @@ import {
 } from './commons';
 import { getOptimizableGlobals } from './optimizer';
 import { createScopeHandler } from './scopeHandler';
-import { rejectImportExpressions } from './sourceParser';
+import { generateSource, parseSource, rejectImportExpressions } from './sourceParser';
```

Comment:
Could you resolve conflicts with current version? For example, from https://github.com/Agoric/realms-shim/commit/3d47d5e806265a1f66f6235dac4efee9c4679ae6 you'll need to update from `rejectImportExpressions` to `rejectDangerousSources`. Probably they are all trivial like this, but it would help review. Thanks.